config/print/main.go

```diff
@@ -66,4 +68,58 @@ func main() {
 	}
 
 	fmt.Printf("%s\n", b)
+
+	if opt.printFieldUse {
+		err = printFieldNames(b)
+		if err != nil {
+			logrus.WithError(err).Fatal("Error printing fields")
+		}
+	}
+}
+
+func printFieldNames(b []byte) error {
+	type Dashboard struct {
+		DashboardTabs []map[string]interface{} `json:"dashboard_tab"`
+	}
+	type Config struct {
+		TestGroups      []map[string]interface{} `json:"test_groups"`
+		Dashboards      []Dashboard              `json:"dashboards"`
+		DashboardGroups []map[string]interface{} `json:"dashboard_groups"`
+	}
+	var output Config
+	err := json.Unmarshal(b, &output)
```

Comment:
You can avoid a marshal/unmarshal by using the "cfg" on line 61 instead of the JSON byte array.

Reply:
I'm not sure I can avoid it, since this weird workaround needs to marshal into a particular struct to make counting the fields possible? (Might be missing something though)

Reply:
Ah, I see. You could get the field names with the reflect library to avoid the serialize/deserialize, but I can see the argument that this is also a clear and good way of doing this.

/approve
/lgtm